packages/components/src/templates/next/components/internal/Link/utils/getTextContent.ts

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+
+export const getReactNodeText = (node: React.ReactNode): string => {
+  let text = ""
+
+  React.Children.forEach(node, (child) => {
+    if (child === null) return
+    if (typeof child === "boolean") return
+    if (typeof child === "undefined") return
+
+    if (typeof child === "string" || typeof child === "number") {
+      text += child.toString()
```

Comment:
[issue] i think we should make `text` a string and join later. this is because if we hvae `string` or `number` child here, this would jsut add the raw string to the `text` **wihtout spaces**.

as an example, if we have 2 children: `[hello, world]` - this returns `"helloworld"`

Reply:
fxed!